clientd3d/config.c

```diff
@@ -294,9 +299,23 @@ void ConfigLoad(void)
    config.active_stat_group = GetConfigInt(misc_section, INIActiveStatGroup, 5, ini_file);
 
    // Determine if we should be using gpu efficiency mode or not.
-   char config_value[255];
-   GetPrivateProfileString("config", "gpuefficiency", "error", config_value, 255, "./config.ini");
-   config.gpuEfficiency = (0 == strcmp(config_value, "true"));
+   auto one_time_gpu_efficiency_enablement = GetConfigInt(config_section, 
+	   "gpuefficiencyonetimeflip", False, config_ini_file);
+   if (!one_time_gpu_efficiency_enablement)
+   {
+      // Perform one-time flip to enable GPU efficiency mode regardless of current preference.
```

Comment:
Add a TODO with the date you added it, mentioning that this should be removed after the next update (since essentially all existing users will presumably run it after that).